src/Utils.sol

```diff
@@ -1,16 +1,18 @@
 
 // SPDX-License-Identifier: Apache 2
-pragma solidity ^0.8.13;
+pragma solidity ^0.8.19;
 
-import "./interfaces/IWormholeRelayer.sol";
+error NotAnEvmAddress(bytes32);
 
-function toWormholeFormat(address addr) pure returns (bytes32) {
-    return bytes32(uint256(uint160(addr)));
+function toUniversalAddress(address addr) pure returns (bytes32 universalAddr) {
```

Comment:
I was wondering too, but decided against it.

Kinda goes hand in hand with the point you raised in the topmost comment about breaking changes, changelog, etc.

I don't quite know how to strike the balance here, at least in part because I don't know who's actually using this repo.

For now, I'd lean towards just using (semantically) versioned tags and focusing on making the repo as useful as possible, rather than worrying too much about breaking things. Strong growth first, maturation and professionalization after. But :man_shrugging:

Reply:
Sounds good, let's start with a 0.x versioning scheme and be liberal with breaking changes. Then we can focus on stabilising the API. I propose tagging the current main HEAD as `0.1`, then these new changes as `0.2`